CODEOWNERS

```diff
@@ -30,6 +30,7 @@ Makefile*                                                        @istio/wg-test-
 /pkg/envoy/                                                      @istio/wg-networking-maintainers
 /pkg/istio-agent/                                                @istio/wg-networking-maintainers @istio/wg-security-maintainers
 /pkg/keepalive/                                                  @istio/wg-networking-maintainers
+/pkg/monitoring/                                                 @istio/wg-networking-maintainers @istio/wg-policies-and-telemetry-maintainers
```

Comment:
I think this should be `@wg-test-and-release-maintainers @istio/wg-policies-and-telemetry-maintainers`? The old pkg repo PRs were approved by T&R maintainers.